core/chains/evm/client/simulated_backend_client.go

```diff
@@ -155,26 +184,26 @@ func init() {
 	var err error
 	balanceOfABI, err = abi.JSON(strings.NewReader(balanceOfABIString))
 	if err != nil {
-		panic(errors.Wrapf(err, "while parsing erc20ABI"))
+		panic(fmt.Errorf("%w: while parsing erc20ABI", err))
```

Comment:
We use it all over core because it predates the latest error wrapping features of Go. This is just making a small change in a single file to bring the code up to date with latest Go standards.

Reply:
ahh I didn't realize that. Thanks for the explanation :)

Reply:
I did research this one a bit just to be sure so I'll add a bit more info here for anyone else interested in why this change.

- `github.com/pkg/errors` is in maintenance mode because of the latest Go standard of wrapping errors
- using `errors.Is` from the above package simply wraps the Go standard `errors.Is` so threat from not using `Wrap`
- `go-ethereum` returns errors in the standard format, so continuing that pattern in our own simulated client makes the most sense